com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +31,142 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
 
+## Creating your own Input Axis Controller
+
+The default implementation of `CinemachineInputAxisController` can process input from the Input package and from Unity's legacy input system. In the case where you want your input to come from an other source, you may need to create your own input controller.
+
+```cs
+using Unity.Cinemachine;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+[Serializable]
+public class SliderReader : IInputAxisReader 
+{
+    // Replace this with the type you would like to use
+    [SerializeField]
+    Slider m_Slider; 
+
+    public float GetValue(UnityEngine.Object context, 
+        int playerIndex, 
+        bool autoEnableInput, 
+        IInputAxisOwner.AxisDescriptor.Hints hint)
+    {
+        if (m_Slider != null)
+            return m_Slider.value;
+        
+        return 0;
+    }
+}
+
+//The component that you will add to your CinemachineCamera.
+public class SliderInputController : InputAxisControllerBase<SliderReader> {} 
+
+// Optional but recommended to display a nice inspector.
+#if UNITY_EDITOR
+[UnityEditor.CustomEditor(typeof(SliderInputController))]
+public class SliderControllerEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+## Using PlayerInput and PlayerInputManager components with Cinemachine
+
+For more complex input configuration like supporting multiple devices, local multiplayer... You will need to receive inputs from the `PlayerInput` component provided by the Input System package. This section is assuming you already know how to setup this component. All the documentation can be found on the Input System package documentation page and samples.
+
+### Read from PlayerInput component
+
+To read values from a `PlayerInput` with a `behaviour` set to `InvokeCSharpEvents`, You need to create a custom `InputAxisController` that subscribes to `onActionTriggered`. The example bellow shows how to receive and wire those inputs accordingly. Add this script to your `CinemachineCamera` and assign the `PlayerInput` field.
+
+```cs
+using UnityEngine;
+using Unity.Cinemachine;
+using System;
+using UnityEngine.InputSystem;
+
+class PlayerInputReceiver : InputAxisControllerBase<SimpleReader>
+{
+    [SerializeField]
+    PlayerInput m_PlayerInput;
+
+    void Awake()
+    {
+        // Call back when the PlayerInput receives an Input.
+        m_PlayerInput.onActionTriggered += OnActionTriggered;
+    }
+
+    public void OnActionTriggered(InputAction.CallbackContext value)
+    {
+        // Sends the Input to all the controllers.
+        for (var i = 0; i < Controllers.Count; i ++)
+        {
+            Controllers[i].Input.SetValue(value.action);
+        }
+    }
+}
+
+[Serializable]
+class SimpleReader : IInputAxisReader
+{
+    // Assumes the action is a Vector2 for simplicity but can be changed for a float.
+    Vector2 m_Value;
+    [SerializeField]
+    InputActionReference m_Input;
+    [SerializeField]
+    float m_Gain;
```

Comment:
should init to 1, so input works by default

Reply:
fixed